content/Events/_index.md

```diff
@@ -1,8 +1,20 @@
 ---
 title: Events 
-description: Events in the Kubernetes community
-calendar: true
+description: Kubernetes Contributor Summits 
 ---
-{{< calendar >}}
 
-See it on the web at [calendar.google.com](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles) , or paste this [iCal url](https://calendar.google.com/calendar/ical/cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com/public/basic.ics) into any [iCal client](https://en.wikipedia.org/wiki/ICalendar). Do NOT copy the meetings over to a your personal calendar, you will miss meeting updates. Instead use your client's calendaring feature to say you are attending the meeting so that any changes made to meetings will be reflected on your personal calendar. 
\ No newline at end of file
+# What ?
```

Comment:
```suggestion
```
I'd just remove this altogether -- The text below will be a good enough intro.

Reply:
Sure